examples/cluster-basic.yaml

```diff
@@ -30,6 +30,7 @@ spec:
     httpLoadBalancing: false
     networkPolicyConfig: false
     horizontalPodAutoscaling: false
```

Comment:
Having a tough time finding info on this one at cluster level or in console. We previously defaulted to enabled, can't tell what current default is.

Reply:
It's documented here https://cloud.google.com/kubernetes-engine/docs/how-to/horizontal-pod-autoscaling it seems it's only possible to create an autoscaler for a workload that's running on the cluster after it's created. The SDK documentation on it https://pkg.go.dev/google.golang.org/api/container/v1#HorizontalPodAutoscaling says that "When enabled, it ensures that metrics are collected into Stackdriver Monitoring." so I guess this is more about enabling monitoring for the feature than enabling the feature. It's existence here predates me, maybe it comes from the old KE driver and maybe it was more prominently featured in the console back then.

Reply:
Seems like an odd way to approach HPA. I guess just leave for now.